security/deployment.adoc

```diff
@@ -0,0 +1,85 @@
+[[security-deployment]]
+= Deployment
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-deployment-what-can-be-deployed]]
+== What Can Be Deployed in a Container?
+
+In case anything falls through during the build process, or for situations where
+a vulnerability is discovered after an image has been deployed, you want yet
+another layer of security: tooling for automated, policy-based deployment. This
+is particularly important as best practice is not to patch running containers,
+but instead use triggers to rebuild and replace with automated updates.
+
+For example:
+
+. An application built using three container image layers: core, middleware, and
+the application layer.
+. An issue is discovered in the core image and that image is rebuilt.
+. Once the build is complete, the image is pushed to the OpenShift Container Registry.
+. {product-title} detects that the image has changed. For builds that are
+dependent on this image and that have triggers defined, {product-title}
+automatically rebuilds and deploys the application image, incorporating the
+fixed libraries and ensuring that the code running in production is always
+identical to the most recently updated image.
+
+All of these capabilities work together to integrate security capabilities into
+your CI/CD process and pipeline.
+
+image::secure_deployments.png["Secure Deployments", align="center"]
+
+[[security-deployment-sccs]]
+== Security Context Constraints (SCCs)
+
+In addition, {product-title} has built-in _security context constraints_ (SCCs)
+that you can use to define a set of conditions that a pod (a collection of
+containers) must run with in order to be accepted into the system.
+
+SCCs allow an administrator to control the:
+
+- Running of privileged containers. A key security concept is that processes
+should be run with the least possible privilege; this is true for containers as
+well. You do not want your containers to run as root unless absolutely
+necessary. SCCs include a policy that enforces "if a container requires root
+access, you cannot deploy it."
+- Capabilities a container can request to be added.
+- Use of host directories as volumes.
+- SELinux context of the container.
+- Container user ID.
```

Comment:
Thanks, I'll make it more clear it's a partial list and that more are possible / direct them to the Further Reading section for more.